finetune/base.py

```diff
@@ -57,8 +58,8 @@ def cleanup():
 
         atexit.register(cleanup)
 
-        self.config = config or get_default_config()
-        self.config.update(kwargs)
+        self.config = get_config(**kwargs)
```

Comment:
I don't understand why this changed?

Reply:
It doesn't need to be, I should revert this.  At the very least this change should be in a separate PR.